plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/config/Settings.java

```diff
@@ -0,0 +1,11 @@
+package io.jenkins.tools.pluginmanager.config;
+
+import java.io.File;
+
+public class Settings {
```

Comment:
I am not sure such defaults are sustainable. They might work for Docker images, but not in other environments

Reply:
Ok. I think we'd need to have more discussions for me to better understand how to fix this. Created Jira ticket: https://issues.jenkins-ci.org/browse/JENKINS-58076